solc/jsonCompiler.cpp

```diff
@@ -218,12 +218,13 @@ string compile(StringMap const& _sources, bool _optimize, CStyleReadFileCallback
 		{
 			// Do not taint the internal error list
 			ErrorList formalErrors;
-			if (compiler.prepareFormalAnalysis(&formalErrors))
+			ErrorReporter errorReporter(formalErrors);
+			if (compiler.prepareFormalAnalysis(&errorReporter))
 				output["formal"]["why3"] = compiler.formalTranslation();
-			if (!formalErrors.empty())
+			if (!errorReporter.errors().empty())
```

Comment:
Since you pass in `formalErrors` to the reporter, these two instances could just refer to `formalErrors`?